spanner/tests/system.py

```diff
@@ -531,6 +536,60 @@ def test_read_w_manual_consume(self):
         self.assertEqual(streamed._current_row, [])
         self.assertEqual(streamed._pending_chunk, None)
 
+    def test_read_w_index(self):
+        ROW_COUNT = 2000
```

Comment:
I get it, I'm not trying to be pedantic, I'm trying to be practical. This is **not** how people use `ALL_CAPS`.